extension/test/e2e/credit-card-amount-update.spec.js

```diff
@@ -4,11 +4,7 @@ import { routes } from '../../src/routes.js'
 import config from '../../src/config/config.js'
 import CreditCardInitSessionFormPage from './pageObjects/CreditCardInitSessionFormPage.js'
 import httpUtils from '../../src/utils.js'
-
-import {
-  waitUntil,
-  fetchNotificationInterfaceInteraction,
```

Comment:
You can use `fetchNotificationInterfaceInteraction` in test-utils.js instead of implementing another one inside e2e test

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1079/commits/4236bccb868d1fc73d888c910dc411b38bf6f671